frontend/src/app/utils/format.ts

```diff
@@ -1,25 +1,39 @@
 export function formatNumber(value: number, showCurrencySymbol = false, shrink = false,
-  accuracy = shrink ? 1 : 2): string {
+  accuracy = 1): string {
```

Comment:
For backward compatibility. Before this patch only percentage display overrides this explicitly to 1. So all numbers are effectively displayed like this